### PR TITLE
Feature preventive offline

### PR DESCRIPTION
**Pull request by ironoa**

### CODE REVIEW

**Comment by ironoa** [comment]
w3f/infrastructure#172

File: .circleci/config.yml [hunk 1]
```diff
@@ -42,6 +42,27 @@ jobs:
           command: |
             /scripts/integration-tests.sh
 
+  testPrometheusRules:
+    docker:
+      - image: web3f/ci-commons:v2.1.3
```
Comment (fgimenez) [inline .circleci/config.yml:47]: We can bump to the latest version
```suggestion
      - image: web3f/ci-commons:v2.4.0
```

File: test/prometheus/alertrules.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+          - eval_time: 11m # Values: 0 1 1 1 1 1 1 1 1 1 1
+            alertname: ValidatorOfflineSessionLong
+            exp_alerts:
+                - exp_labels:
+                    severity: critical
+                    origin: cluster
+                    name: node0
+                  exp_annotations:
+                      message: "Target node0 has either not authored any block or sent any heartbeat yet in this session"                    
+          - eval_time: 15m # Values: 0 1 1 1 1 1 1 1 1 1 1 1 1 0 0
```
Comment (fgimenez) [inline test/prometheus/alertrules.yaml:37]: Why do we need this alert_rule_test, isn't enough with the previous one?

File: test/prometheus/alertrules.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+rule_files:
```
Comment (fgimenez) [inline test/prometheus/alertrules.yaml:1]: :rocket:

**Pull request merged**